libp2p/NodeTable.cpp

```diff
@@ -674,6 +685,69 @@ std::shared_ptr<NodeEntry> NodeTable::handlePingNode(
     return sourceNodeEntry;
 }
 
+std::shared_ptr<NodeEntry> NodeTable::handleENRRequest(
+    bi::udp::endpoint const& _from, DiscoveryDatagram const& _packet)
+{
+    std::shared_ptr<NodeEntry> sourceNodeEntry = nodeEntry(_packet.sourceid);
+    if (!sourceNodeEntry)
+    {
+        LOG(m_logger) << "Source node (" << _packet.sourceid << "@" << _from
+                      << ") not found in node table. Ignoring ENRRequest request.";
+        return {};
+    }
+    if (sourceNodeEntry->endpoint() != _from)
+    {
+        LOG(m_logger) << "ENRRequest packet from unexpected endpoint " << _from << " instead of "
+                      << sourceNodeEntry->endpoint();
```

Comment:
Strange formatting, why is this expected endpoint on a separate line?

Reply:
That's clang-format wraps around max line width